cypress/e2e/home.cy.ts

```diff
@@ -0,0 +1,8 @@
+const ProjectURL = Cypress.env("BASE_URL");
```

Comment:
We declare variables in `camelCase`

Reply:
Issue solved in commit 645dc18b52e76988cc47099799ba8ffcb7515fc2.